app/gui/enso-profiler-enso-data/src/bin/api_events_to_profile.rs

```diff
@@ -0,0 +1,84 @@
+//! Tool that translates a backend message log into the [`enso_profiler`] JSON format.
+//!
+//! # Usage
+//!
+//! The tool reads a CSV backend message log, and converts it to a
+//! [JSON-formatted event log](https://github.com/enso-org/design/blob/main/epics/profiling/implementation.md#file-format)
+//! for use with [`enso_profiler`] tools.
+//!
+//! For example:
+//!
+//! ```console
+//! ~/git/enso/enso_data $ cargo run --bin api_events_to_profile < messages.csv > messages.json
+//! ```
```

Comment:
We have several profiling tools / binaries now. I think it would be beneficial to integrate them with the product / tooling (like our run script). Instead of remembering about all the tools and how to use them, they should be integrated into commands of our application. This does not need to be part of this PR, but I would love to discuss it during the next performance team meeting. CC @sylwiabr @MichaelMauderer @kazcw

Reply:
Those tools have no need to use node or JS, so it would be great if they stay available as standalone commands that only require rust/cargo to run. This does  not preclude allowing them to be run through the run script, but I do not see a big need for that and would not prioritize this. I would rather see a nice document that summarizes our tools and documents their use cases and how to use them.